Governance.md

```diff
@@ -0,0 +1,186 @@
+# Introduction
+This document suggests a governance model to be used by the Neo core developers. The goal is to develop a fair model that balances compensation with each individual's responsibilities and outcomes/deliverables. 
+This document also suggests the creation of a Core Developer Treasury to maintain the core developer team. 
+
+## Main Issue
+Developing a fair plan that balances compensation with each individual's responsibilities and outcomes/deliverables is essential. Stability, fairness, and organization are required to attract and retain talent.
+
+### Existing Problems
+We face challenges in our development processes, for example:
+- Issues raised that are not relevant to the current objectives of the project. 
+- Abandoned issues or PRs.
+- Evaluating results based only on PRs.
+  - Leading to competition instead of collaboration. 
+- A feeling of disorganization.
+- Unclear assignment of responsibilities.
+  - Who responds to people?
+  - Who verifies results?
+  - Who does the release?
+  - Who details the issues?
+  - ...
+- Lack of processes leading to poor efficiency
+- No overall vision of the code base health.
+
+
+It looks like, culturally speaking, we are failing to understand this:
+**Everything is about people. Software is about people, not code.**
+
+### Understanding People:
+- People want to work in a professional and welcoming environment.
+- People want to feel safe.
+- People want challenges that match their skills.
+- People want a sense of professional growth.
+- People want to create a legacy.
+- People are not the same: different people provide different value
+- People want a fair work environment, including fair compensation.
+- People need a sense of belonging.
+- People don't want and can't be controlled. They need to be convinced.
+- People don't like feeling being used.
+- People don't like being ignored.
+- People like privacy.
+- Seasoned developers are older people who demand more stability. 
+- People want to see the results of their work.
+- We must understand the people using our stuff and how they feel about it.
+  - Identify usability improvement points (installation, configuration, etc.).
+- We want to attract and retain the best people.
+- People prefer teams that work online and asynchronously.
+  - Culture compatible with everyone's reality, timezone, etc.
+- We want people to deliver their best.
+- People expect a long-term vision.
+- We want people that aim for the overall network improvement.
+- People work better with incentives than punishments.
+- People make mistakes, but no one should be negligent or lack general responsibility.
+
+### Proposed Solution: Create a clear Core Developer Team and Treasury
+This governance model should embrace both responsibilities and compensations. This relates, but it's not the same as the development process #2955.
```

Comment:
you have to use a link or else people when they download this file won't be able to to click that issue.